frontend/conf/application.conf

```diff
@@ -1,6 +1,8 @@
 # Config for the 'Vanilla' Membership site
 include "copy"
 
+play.application.loader = "wiring.AppLoader"
```

Comment:
I put the subs one in the root package and called it MyApplicationLoader (not sure that was ideal naming, but it came from the play exaple docs)

Reply:
I copied this from support-frontend, which copied it from somewhere else that I can't remember.